README.md

```diff
@@ -363,3 +363,56 @@ To test if the setup is working, you should do the following with an image that
 4. Images uploaded to the S3 bucket should be publicly available because of the defined policy.
 
 5. Accessing an image on S3 that does not exist should cause S3 to redirect back to the website.
+
+# Setup Cloudflare (optional)
+
+The bundle supports the purging of thumbnails from Cloudflare CDN. This is handled by the `CloudflareManager` and is
+disabled by default. To enable it, do the following.
+
+* Add to your `config.yml`:
+
+    ```
+    genj_thumbnail:
```

Comment:
that configuration should be part of the cdn.yml
so that we can separate _production_ from _development_ easier.

Reply:
This is changed